### PR TITLE
Finished Lab6

### PR DESCRIPTION
**Pull request by Sqaiser**

### CODE REVIEW

**Comment by Sqaiser** [comment]
Professor please grage my lab.
Thank you

**Comment by khatchad** [comment]
There seems to be a bunch of files missing here. I'll grade what I see.

File: src/cst1201/Bowtie.java [hunk 1]
```diff
@@ -1,10 +1,48 @@
...
 
+import cst1201.Turtle;
+
 /**
  * A class representing a Bowtie drawn by a Turtle object.
  *
- * @author (your name)
+ * @author Sameen Qaiser
  */
 public class Bowtie {
```
Comment (khatchad) [inline src/cst1201/Bowtie.java:10]: :-1: 

> [2pt] Your Bowtie class has a draw() method that draws a Bowtie

File: src/cst1201/Bowtie.java [hunk 2]
```diff
@@ -1,10 +1,48 @@
...
+  
+
+       
+
+       
+
+    }
 
 }
+
```
Comment (khatchad) [inline src/cst1201/Bowtie.java:48]: :-1: 

> [1pt] You have created a new InitialDrawing class with a draw() method

Comment (khatchad) [inline src/cst1201/Bowtie.java:48]: :-1: 

> [2pt] Your InitialDrawing class can draw your initials wth a plus sign

Comment (khatchad) [inline src/cst1201/Bowtie.java:48]: :-1: 

> [1pt] You have created a new HouseDrawing class with a draw() method

File: src/cst1201/Bowtie.java [hunk 3]
```diff
@@ -1,10 +1,48 @@
...
  *
- * @author (your name)
+ * @author Sameen Qaiser
  */
 public class Bowtie {
+    private static Object myTurtle;
+ 
+    public static void main(String[] args) {
+         
+         Turtle turtle = new Turtle();
```
Comment (khatchad) [inline src/cst1201/Bowtie.java:15]: :-1: 

> [1pt] Good programming style -- including proper indentation is expected

**Pull request closed without merging**